spec/spec_helper.rb

```diff
@@ -88,7 +89,11 @@
       ex.run
     end
   end
-
+  
+  config.after do |example|
+    save_and_open_page if example.metadata[:type] == :feature and example.exception.present?
```

Comment:
I'm pretty focused on the test results, rather than wanting to see the page every time there's a failure.

Is there a way to enable this feature so that people can use it if they want? For example, if you were working on an error and couldn't understand the problem from the test output, you could add this to the test yourself?

Having it on all the time in the app seems a bit prescriptive to me—mainly because it takes my focus to a different app when I want to look at the test failure message.

Thoughts @snoblenet ?